aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsProxyHttpServletRequest.java

```diff
@@ -529,7 +529,10 @@ public String getParameter(String s) {
     @Override
     public Enumeration<String> getParameterNames() {
         List<String> paramNames = new ArrayList<>();
-        paramNames.addAll(request.getQueryStringParameters().keySet());
+        Map<String, String> queryStringParameters = request.getQueryStringParameters();
```

Comment:
Could we reduce this to:
if (request.getQueryStringParameters() != null) {
  paramsNames.addAll(request.getQueryStringParameters().keySet());
}